lib/entice/logic/MapInstance.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Entice.Logic.MapInstance do
+  alias Entice.Entity
+  alias Entice.Logic.MapInstance
+
+
+  defstruct(players: %{},
+    npcs: %{},
+    map: nil)
+
+  def start_instance(map, player_entities \\ [], npc_info \\ []) do
```

Comment:
personally, i'd say lets put this all into register and not create the entity ourselfes...
thoughts?